css/header.css

```diff
@@ -0,0 +1,69 @@
+:root {
+    --red: rgb(175, 77, 41);
+}
```

Comment:
Its good to see you defining a variable but it would be great to see more, maybe even some fonts!